### ISSUE TITLE
[7.0.5] LDAP logs many errors

### ISSUE THREAD

**Issue by cdamken** [owncloud/core#15678]
We've got tons of errors like this one:
```
{user_ldap} Error when searching: Can't contact LDAP server code -1
```
However, the ldap servers seem to be running fine. I I don’t have any users reporting, that they can’t log in.

——

The ldap servers are logging stuff like this:

```
Mar 29 20:06:50 drive-ldap1 slapd[2072]: connection_read(15): no connection!
Mar 29 20:07:02 drive-ldap1 slapd[2072]: connection_input: conn=23839114 deferring operation: binding

```
But from what I can tell, those errors are unrelated.
- The first one means, that the client is disconnecting improperly (no unbind)
- The second one means that a bind operation was canceled by the client. (I know it’s not what it says, but that is the conclusion after googling for a while)

——

We are running Owncloud version 7.0.5

Any Ideas?

@MorrisJobke @blizzz 

00002799

**Comment by MorrisJobke** [comment]
I thought @blizzz already answered this. But this is the only issue with this error message.

Can we do anything against this message? Looks like the connection to the LDAP server is somehow instable.

**Comment by cdamken** [comment]
@MorrisJobke @blizzz I suppose yes, I have another issue that must be related with this: https://github.com/owncloud/client/issues/2848

**Comment by blizzz** [comment]
@cdamken can you confirm they have a proxy or load balancer instead of talking directly to one LDAP?

**Comment by cdamken** [comment]
@blizzz We have installed both a load balanced LDAP server and a backup server as well. (As @felixboehm would have knows). The load balanced LDAP caused all the problems we had in the first place, so it has been replaced with a  direct connection to the primary LDAP and heavy caching to keep the load the LDAP server in acceptable limits.

**Comment by blizzz** [comment]
So, the reason for the message is that connections to LDAP get lost. With https://github.com/owncloud/core/pull/15332 we will throw an Exception instead, as it can have other side effects as well.

**Comment by MorrisJobke** [comment]
@cdamken Just for your info: The fix is already in the code base for 8.1 and 7.0.6 and a proposed PR is ready for review for the 8.0.4 release

**Comment by DeepDiver1975** [comment]
We can close this issue from my understanding - the error message itself are valid and a fix for the explicit error condition is in place.
